uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -86,6 +86,12 @@ function _hrleaveandabsences_create_administer_menu_tree($leaveAndAbsencesAdminN
           'name'       => 'leave_and_absence_manage_work_patterns',
           'url'        => 'civicrm/admin/leaveandabsences/work_patterns?action=browse&reset=1',
           'permission' => 'administer leave and absences',
+      ),
+      array(
+          'label'      => ts('General Settings'),
+          'name'       => 'leave_and_absence_general_settings',
+          'url'        => 'civicrm/admin/leaveandabsences/generalsettings',
```

Comment:
s/generalsettings/general_settings/

I know it's a bit odd, because leaveandabsences is all together, but it's just to keep "consistency" with work_patterns and public_holidays.